algebird-test/src/test/scala/com/twitter/algebird/ScanTest.scala

```diff
@@ -66,7 +56,7 @@ object ScanTest {
 
 }
 
-class ScanTest extends WordSpec with Matchers with GeneratorDrivenPropertyChecks {
+class ScanTest extends WordSpec with Matchers with ScalaCheckDrivenPropertyChecks {
```

Comment:
are these tests actually running? the code coverage now things you have zero coverage.

I wonder if there is something about the new scalatest that is broken here.

If you break your code and do `test` locally, does it still pass?

Reply:
@johnynek  You're right that the tests weren't running (embarrassing!), but it wasn't using the more up-to-date-import, it was worse than that: when I wrote a method for the new properties for the tests on that commit, a missing close-curly accidentally made the entire test suite exist inside of the method, which was then never being executed (even more embarrassing!).

Is there a way to force CI to give a code coverage report on an adhoc basis, or should I just wait until it auto-runs?

Reply:
if you look here you can see how the coverage report is generated:

https://github.com/twitter/algebird/blob/develop/.travis.yml#L14

basically you do something like `sbt coverage clean test coverageReport`, so you are enabling coverage instrumentation with `coverage` then after you run `test` you run `coverageReport` and it emits an xml and html output you can load in your browser (it prints the path IIRC).